contrib/zenith/libpagestore.c

```diff
@@ -71,7 +71,7 @@ zenith_connect()
 	{
 		PGresult   *res;
 
-		query = psprintf("callmemaybe %s %s", zenith_timeline, callmemaybe_connstring);
+		query = psprintf("callmemaybe %s %s %s", zenith_tenant, zenith_timeline, callmemaybe_connstring);
```

Comment:
let's also check that zenith_tenant and zenith_timeline are non-null. On some platfroms printf() instead of writing '(NULL)' would just crash.